test/extensions/quic_listeners/quiche/platform/epoll_address_test_utils_impl.h

```diff
@@ -0,0 +1,42 @@
+#pragma once
+
+// NOLINT(namespace-envoy)
+
+// This file is part of the QUICHE platform implementation, and is not to be
+// consumed or referenced directly by other Envoy code. It serves purely as a
+// porting layer for QUICHE.
+
+#include <sys/socket.h>
+
+#include <algorithm>
+
+#include "envoy/network/address.h"
+
+#include "test/test_common/environment.h"
+
+namespace epoll_server {
+
+namespace {
+
+int addressFamilyUnderTestHelper() {
+  std::vector<Envoy::Network::Address::IpVersion> versions =
+      Envoy::TestEnvironment::getIpVersionsForTest();
+  if (std::find(versions.begin(), versions.end(), Envoy::Network::Address::IpVersion::v4) !=
+      std::end(versions)) {
```

Comment:
nit:  How about we either use versions.(begin|end), or std::(begin|end)(versions), but not a mix of them?

Reply:
done